client/jetpack-cloud/sections/settings/advanced-credentials/host-info.ts

```diff
@@ -182,6 +180,136 @@ export const topHosts: Host[] = [
 					},
 				],
 			},
+			user: {
+				ftp: [
+					{
+						type: InfoTypes.OrderedList,
+						items: [
+							translate( 'Login to your Bluehost cPanel' ).toString(),
+							translate(
+								'Click the "Advanced" tab towards the left side of the account.'
+							).toString(),
+							translate(
+								'Choose FTP from the sub-menu, or click the FTP Accounts icon from the Files section.'
+							).toString(),
+							translate( 'Create a new FTP account' ).toString(),
+						],
+					},
+					{
+						type: InfoTypes.Text,
+						text: translate(
+							'Your Bluehost username will end with your domain (e.g. test@example.com).'
+						).toString(),
+					},
+					{
+						type: InfoTypes.Line,
+					},
+					{
+						type: InfoTypes.Link,
+						text: translate( 'Vist my Bluehost cPanel' ).toString(),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **12** times:
`translate( 'Visit my Bluehost cPanel' )` *ES Score: 21*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/jetpack-cloud-update-new-credentials-styles)